tests/test_main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.main import list_reverse
+
+
+def test_final_result():
+    assert list_reverse(['Mate', 'Academy']) == ['ymed', 'acAetaM']
+
+
+def test_is_element_length_correct():
+    origin_list = ['I', 'am', 'a', 'student!']
+
+    for i, j in zip(origin_list, list_reverse(['I', 'am', 'a', 'student!'])):
```

Comment:
You should provide the same fix for all tests:
```
list_reverse(['I', 'am', 'a', 'student!']) == <expected value>
```

Reply:
Fixed. I thought, previous code was more readable.

Reply:
You have not fixed this test
The test should be similar to the first:
```
assert list_reverse(['Mate', 'Academy']) == ['ymed', 'acAetaM']
```
But with other input and expected value.

Reply:
So you don't need `for`, `zip` or any other extra functions. They make tests unclear